django_filters/filters.py

```diff
@@ -220,6 +220,11 @@ class DateRangeFilter(ChoiceFilter):
         4: (_('This year'), lambda qs, name: qs.filter(**{
             '%s__year' % name: now().year,
         })),
+        5: (_('Yesterday'), lambda qs, name: qs.filter(**{
+            '%s__year' % name: now().year,
+            '%s__month' % name: now().month,
+            '%s__day' % name: (now().day - timedelta(days=1)).day,
```

Comment:
Should this be `(now().day - timedelta(days=1)).day` or `(now() - timedelta(days=1)).day`?

I don't think it will make a (significant) difference in most cases, but I think it might be a bit clearer in the long run.